spring-cloud-stream-module-launcher/src/main/java/org/springframework/cloud/stream/module/launcher/ModuleLauncher.java

```diff
@@ -59,26 +62,35 @@ public ModuleLauncher(ModuleResolver moduleResolver) {
 	/**
 	 * Launches one or more modules, with the corresponding arguments, if any.
 	 *
+	 * Modules must be passed in "natural" left to right order, and will be deployed in reverse order
```

Comment:
Should not mention deploying in reverse order here - at this level, that is an implementation detail. ModuleLauncher will encapsulate these modules as a single deployment unit.